dts/bindings/mbox/andestech,plic-sw.yaml

```diff
@@ -1,20 +1,19 @@
-
 #
 # Copyright (c) 2022, Andes Technology Corporation.
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 
-description:
-    This is a representation of AndesTech PLIC-SW node
+description: |
+  This is a representation of AndesTech PLIC-SW node
 
 compatible: "andestech,plic-sw"
 
 include: [base.yaml, mailbox-controller.yaml]
 
 properties:
-    reg:
-      required: true
+  reg:
+    required: true
```

Comment:
these fixes should go in the previous commit.

Reply:
Oh, it's my negligence.
I have update the source code.

Thank you.